test/atm/global-workflow/jjob_ens_final.sh

```diff
@@ -46,7 +46,7 @@ fi
 
 # Execute j-job
 if [ $machine = 'HERA' -o $machine = 'ORION' ]; then
-    sbatch --ntasks=1 --account=$ACCOUNT --qos=debug --time=00:10:00 --export=ALL --wait ${HOMEgfs}/jobs/JGLOBAL_ATMENS_ANALYSIS_FINALIZE
+    sbatch --ntasks=1 --account=$ACCOUNT --qos=batch --time=00:10:00 --export=ALL --wait ${HOMEgfs}/jobs/JGLOBAL_ATMENS_ANALYSIS_FINALIZE
```

Comment:
The numerous locations in which `debug` had to be changed to `batch` highlights the need to specify the queue in central location.   The same comment applies to `ACCOUNT`.   This can be the subject of a new issue and PR.